README.md

```diff
@@ -81,48 +80,44 @@ You may optionally provide an object with a `sampleRate` key, defaults to 44100.
 
 ## crunker.fetchAudio(songURL, anotherSongURL)
 
-Fetch one or more audio files.  
```

Comment:
These are line breaks, not 'extra spaces'  ╰（‵□′）╯

Reply:
Wow, you're right, let me fix it. Sorry about that